### ISSUE TITLE
How to refresh LineChart graph when data remain same but xAxisMinimum/maximum changes

### ISSUE THREAD

**Issue by angelos3lex** [wuxudong/react-native-charts-wrapper#377]
I have LineChart with dates in xAxis, and i want to change xAxisMinimun and xAxisMaxium dynamically. 
When I do so, there are two options: 
First, data are also updated->since new xAxisMinimum entered new data will be pushed-removed from data array. 
second: data remain the same as not a single new data was added or removed, because even if I changed the dates, no more/less data existed in these dates.
For example, if I have data from 13-20 August, and xAxisMinimum is 13 of Aug(in timestamp) and max is 20 of Aug(in timestamp), all data will be shown correctly. 
->>> if I change xAxisMinimum to 11 while I had 13, then data remain the same(since I didn't have data before 13 of Aug). But I change xAxisMinimum, so I want them to update the graph, even if data aren't changed, because they should shift a bit to the right and not display data in 11 and 12 of Aug. 
->>>BUT, if I change xMin to 14 (while I had 13), to be 14 of Aug, graph will be refreshed and data of 13 of Aug won't be shown, also xAxis will be refreshed accordingly, which is the correct thing to do.

So, graph is refreshed only if new data are added/removed, when they remain the same it isn't, even if new axis minimum and maximum are passed to xAxis. 

One solution of this, but only for ANDROID, is to call a setState({data:[]}), with empty data just before the setState of real data. Then graph is first rendered with empty data, and then updated with real data. But I do not know why this solution is not working in IOS. Maybe because of the speed of setStates as they are asynchronous? I have no idea. 

So, in conclusion, is there any method like updateGraphEvenIfDataRemainTheSame or something like emptyGraphBeforeActualLoadOfData?

**Comment by angelos3lex** [comment]
I have a temporary solution by pushing in my dataSets the following (though it is kind of hack which I don't like):
```
values: [
        {
          y: -1,
          x: this.props.xMin
        },
        {
          y: -1,
          x: this.props.xMax
        }
      ],
      label: "",
      config: {
        drawValues: false,
        drawCircles: false,
        drawCircleHole: false
      }
```
PS: xAxisMinimum and xAxisMaximum also are set to this.props.xMin and this.props.xMax in my case.
